sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2658,17 +2658,17 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       sql("select LTRIM(BOTH 'S' FROM 'SS abc S')").head
     }
     assert(ae1.getMessage contains
-      "The specified function ltrim doesn't support with option BOTH")
+      "The specified function LTRIM doesn't support with option BOTH")
     val ae2 = intercept[ParseException]{
       sql("select RTRIM(TRAILING 'S' FROM 'SS abc S')").head
     }
     assert(ae2.getMessage contains
-      "The specified function rtrim doesn't support with option TRAILING")
+      "The specified function RTRIM doesn't support with option TRAILING")
     val ae3 = intercept[ParseException]{
-      sql("select TRIM(WINDOW 'S' FROM 'SS abc S')").head
+      sql("select TRIM(OVER 'S' AND 'SS abc S')").head
     }
     assert(ae3.getMessage contains
-      "The specified function trim doesn't support with option WINDOW")
+      "Literals of type 'OVER' are currently not supported")
```

Comment:
Is this caused by other check? But this is not what we want to cover in the test